### PR TITLE
Add Review/513-4

### PR DESCRIPTION
**Pull request by ggurbet**

### CODE REVIEW

**Review by davidtai — APPROVED** [review]

**Comment by davidtai** [comment]
Is this milestone 3 or 4?

File: Reviews/513 - On-Chain MVPR/4 - KYC Contract/README.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+```sh
+@ggurbet ➜ /workspaces/dao-contracts (7c24f46 ✗) $ make build-all
+cargo build --release --target wasm32-unknown-unknown --quiet --features=wasm --no-default-features -p casper-dao-contracts
+cargo build --release --target wasm32-unknown-unknown --quiet --features=wasm --no-default-features -p casper-dao-erc20
+cargo build --release --target wasm32-unknown-unknown --quiet --features=wasm --no-default-features -p casper-dao-erc721
+```
+
+## Overall Impression of usage testing
+
+Installation using given installation instructions failed to work. After making some small adjustments, reviewer was able to install the contracts. Reviewer suggests updating the README to reflect the installation process properly and also use the default commands used by Casper for consistency in the following milestones.
```
Comment (davidtai) [inline Reviews/513 - On-Chain MVPR/4 - KYC Contract/README.md:124]: Has this been requested of the grantee?

Comment (ggurbet) [reply Reviews/513 - On-Chain MVPR/4 - KYC Contract/README.md:124]: This has been updated in the latest README.

**Comment by ggurbet** [comment]
> Is this milestone 3 or 4?

This review is for Milestone 4. It has been incorrectly labeled as Milestone 3 on CRDAO.

**Pull request merged**